intrinsics_path: "/workspace/data/small_box/intrinsics.json"
extrinsics_path: "/workspace/data/small_box/cam2root.json"
model_path: "/workspace/data/small_box/model.ply"

# Box dimensions
known_dims:
  small_box: [0.340, 0.250, 0.095]
  medium_box: [0.255, 0.155, 0.100]

# Planar detector settings
normal_variance_threshold_deg: 60
coplanarity_deg: 60
min_plane_edge_length: 0.03
min_num_points: 20
knn: 50
voxel_size: 0.01
distance_threshold: 0.005
nb_neighbors: 30
std_ratio: 1.5
min_plane_area: 0.00003
